raving-remoras/bangazon-site
# Rd add payment type

# Description
Given the user is authenticated
When the user clicks on a profile affordance in the navigation bar
And the user clicks on the Add Payment Option hyperlink in their profile
Then the user can fill out the payment option form
And click Submit to save payment option

## Steps to Test Solution

1. Once logged in, go to URL http://localhost:8000/website/customer/add_payment  (link will be in customer page, which hasn't been merged)
2. Try to add a payment type with missing data
3. Add a type with all data and check database to make sure it has the correct info and your customer id
4. Run tests

## Testing


## Documentation

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
